src/sysupdate/sysupdated.c

```diff
@@ -0,0 +1,1914 @@
+/* SPDX-License-Identifier: LGPL-2.1-or-later */
+
+#include "bus-util.h"
+#include "hashmap.h"
+#include "log.h"
+#include "sd-bus.h"
+
+#include "build-path.h"
+#include "bus-error.h"
+#include "bus-get-properties.h"
+#include "bus-label.h"
+#include "bus-log-control-api.h"
+#include "bus-polkit.h"
+#include "common-signal.h"
+#include "discover-image.h"
+#include "env-util.h"
+#include "event-util.h"
+#include "escape.h"
+#include "fd-util.h"
+#include "fileio.h"
+#include "main-func.h"
+#include "memfd-util.h"
+#include "mkdir-label.h"
+#include "os-util.h"
+#include "process-util.h"
+#include "sd-json.h"
+#include "service-util.h"
+#include "signal-util.h"
+#include "socket-util.h"
+#include "string-table.h"
```

Comment:
please place the include lines for our public headers (i.e. those name sd-*.h) first, then add an empty line, and then place the include lines for the private headers. That's how we usually do it.

within the two sections sort alphabetically.

Reply:
to quote the coding style:

> However, to not clutter global includes, and to make sure internal definitions will not affect global headers, please always include the headers of external components first (these are all headers enclosed in <>), followed by our own exported headers (usually everything that’s prefixed by sd-), and then followed by internal headers. Furthermore, in all three groups, order all includes alphabetically so duplicate includes can easily be detected.

Reply:
https://systemd.io/CODING_STYLE/#formatting